web/views/stripe_customer_view.ex

```diff
@@ -0,0 +1,9 @@
+defmodule CodeCorps.StripeCustomerView do
+  use CodeCorps.PreloadHelpers, default_preloads: [:user]
+  use CodeCorps.Web, :view
+  use JaSerializer.PhoenixView
+
+  attributes [:email]
```

Comment:
I've added all the attributes listed in the API docs.

I dod not add the subscriptions, because there is at the moment no relationship between the stripe_customer and the stripe_subscription. There is a relationship between `user` and `stripe_subscription` in that a `stripe_subscription` belongs to a `user` and a `stripe_plan`. A `stripe_customer` also belongs to a `user`. 

Should I update the API docs, or should I update our relationships? I'm pretty sure we mentioned this and decided to go with the relationship being on the `user`, not the `stripe_customer`.

Also, regarding the attributes, should we really render all those attributes? What do we get from it, really? Unless I'm missing something, we don't actually use the data client-side. We could even potentially go as far as completely hiding the `stripe_customer` from the client and just adding some sort of `connected_to_stripe` flag on the user record. I'm really not sure what the best approach is here, really.